OSRMTextInstructions/OSRMTextInstructions.swift

```diff
@@ -15,7 +15,7 @@ protocol Tokenized {
 
 extension String: Tokenized {
     public var sentenceCased: String {
-        return String(characters.prefix(1)).uppercased() + String(characters.dropFirst())
+        return String(self.prefix(1)).uppercased() + String(dropFirst())
```

Comment:
While we wait for CocoaPods/CocoaPods#7134 to be resolved, can we keep both the old and new syntaxes around, guarded by `#if swift(>=4)`?